artiq/frontend/artiq_sinara_tester.py

```diff
@@ -709,6 +709,46 @@ def phaser_led_wave(self, phasers):
                     delay(100*ms)
                 phaser.set_leds(0)
                 delay(100*ms)
+    
+    @kernel
+    def get_phaser_adc(self, phaser, ch, cb):
+        self.core.break_realtime()
+        phaser.init()
+        phaser.write8(PHASER_ADDR_ADC_CFG, 0b0000) # set both channel G = 1
+        delay(1*ms)
+        adc_data = phaser.read_adc_raw(ch)
+        delay(1*ms)
+        voltage = phaser.convert_adc_voltage(adc_data)
+        cb(voltage)
+
+    @kernel
+    def setup_phaser_servo(self, phaser, duc):
```

Comment:
That's unused (and as mentioned in #2775 the servo issue is not yet resolved); don't think that belongs in this PR then